mne/channels/channels.py

```diff
@@ -581,6 +581,16 @@ def anonymize(self):
         .. versionadded:: 0.13.0
         """
         anonymize_info(self.info)
+        # XXX This should not be here, but I don't know where to put it
+        # right now
+        if hasattr(self, 'annotations'):
+            if self.annotations is None:
+                pass
+            else:
+                self.annotations.orig_time = None
+                offset = self.first_samp * (1 / self.info['sfreq'])
```

Comment:
`offset = self.first_samp / float(self.info['sfreq'])`

Reply:
we should be guaranteed already that `info['sfreq']` is a float by our `Info` construction and checking functions, so you can scrap the `float` here